libs/utils/decorate.js

```diff
@@ -308,3 +306,22 @@ export function decorateMultiViewport(el) {
   }
   return foreground;
 }
+
+export function turnAnchorIntoVideo({ src, anchorTag }) {
```

Comment:
`decorateVideo` and `turnAnchorIntoVideo` can be confusing; what about using `decorateAnchorVideo`?